regression/tests/studio/studio_base_test.py

```diff
@@ -94,5 +94,4 @@ def clean_up_function(self):
         textbook_page.delete_all_textbooks()
 
         asset_page.visit()
-        asset_count = asset_page.get_files_count()
-        asset_page.delete_file(list(range(asset_count)))
+        asset_page.delete_file()
```

Comment:
So under this line, you'd see if the file has been deleted (possibly in a `wait_for` statement, but not necessarily...)